presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergConnector.java

```diff
@@ -87,6 +92,7 @@ public IcebergConnector(
         this.tableProperties = ImmutableList.copyOf(requireNonNull(tableProperties, "tableProperties is null"));
         this.accessControl = requireNonNull(accessControl, "accessControl is null");
         this.procedures = requireNonNull(procedures, "procedures is null");
+        this.planOptimizer = planOptimizer;
```

Comment:
nit: Refactoring it to use the `requireNonNull` function? Or could the optimizer be null?

Reply:
done